### ISSUE TITLE
Accessibility on Android doesn't work on second launch

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#10059]
1. Turn on accessibility.
2. flutter run. Verify that you can navigate the app.
3. ^C
4. flutter run. You can't navigate the app.
